roles/java/defaults/main.yml

```diff
@@ -1,8 +1,16 @@
 ---
+java_version: 17.0.14+7
+java_major: "{{ java_version.split('.')[0] }}"
+java_build: "{{ java_version.split('+')[1] | default('1-ea-beta') }}"
+java_ver: "{{ java_version.split('+')[0] }}"
```

Comment:
rename to _core which is a semver nomeclature, just _ver doesn't really help in understand what it's meant to hold

https://semver.org/#backusnaur-form-grammar-for-valid-semver-versions